llvm/include/llvm/BinaryFormat/COFF.h

```diff
@@ -806,7 +806,7 @@ enum Feat00Flags : uint32_t {
   Kernel = 0x40000000,
 };
 
-enum class Arm64ECThunkType : uint8_t {
+enum Arm64ECThunkType : uint8_t {
```

Comment:
Why is this change needed here?

Reply:
It's to make switch statement in `initializeECThunks` possible. Otherwise, there is no implicit conversion from `ulittle32_t` and the switch statement would need an explicit cast. And with that cast, we wouldn't be able to use `default` far the warning. I could probably make it work, but the main motivation for having it in a public header was LLD (in #85936), changing it here felt right.